packages/runtime/src/templates/getApiHandler.ts

```diff
@@ -1,3 +1,6 @@
+// This is a runtime function so assuming we don't want to add
+// an additional dependency on a logger (Winston) for runtime code
+/* eslint-disable no-console */
```

Comment:
My assumption for all of our runtime code is that we don't want to add the extra dependency of `winston` to avoid bundle bloat.

Reply:
Yeah, the new logger should be just for build time, no runtime code for the reason you mention. I do wonder if we can use the system log in handlers we generate?

Reply:
Let's look at that for the next PR. Sounds like a good idea.